Bottom: 2aad87442f67375a1d96704ceb15db2efd6f265d
Top:    e5661e871a8d8518bb5cfcd63df265ce97d73cea
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-07-17 12:58:20 -0500

Refresh of add-buildbot-role

---
diff --git a/roles/buildbot-master/defaults/main.yml b/roles/buildbot-master/defaults/main.yml
index 76f8b393..0b19fb38 100644
--- a/roles/buildbot-master/defaults/main.yml
+++ b/roles/buildbot-master/defaults/main.yml
@@ -76,7 +76,7 @@ hostname: '{{ role_name }}'
 image_args:
   BUILDBOT_UID: '{{buildbot_uid}}'
   BUILDBOT_VERSION: 'v{{buildbot_version}}'
-  BUILDBOT_TIMEZONE: '{{ansible_facts.date_time.tz}}'
+  BUILDBOT_TIMEZONE: 'America/Chicago'
 
 # directory of the image source
 image_dir: '{{ docker_projects_dir }}/buildbot-master'
